pkg/apis/postgres-operator.crunchydata.com/v1beta1/standalone_pgadmin_types.go

```diff
@@ -113,6 +118,15 @@ type ServerGroup struct {
 	PostgresClusterSelector metav1.LabelSelector `json:"postgresClusterSelector"`
 }
 
+type PGAdminUser struct {
+	// Whether the user has admin privileges or not.
+	IsAdmin bool `json:"isAdmin"`
+
+	// The username for User in pgAdmin.
+	// Must be unique in the pgAdmin's users list.
+	Username string `json:"username"`
```

Comment:
Hmm... I thought if it isn't `// +optional` then it's required...

Reply:
Seems like it would be good to come to some sort of official consensus on what we expect in this regard. Looking through our code I see a lot of `+kubebuilder:validation:Required`, a handful of `+required`, and then quite a few fields with no tags whatsoever... @cbandy, do you have a preference?

Reply:
The other day, it took me a bit to explain the relationships between `+optional` and `omitempty`. I'm leaning toward "explicit" today; more `+kubebuilder` markers for more clarity.

`+required` doesn't appear on [this page](https://book.kubebuilder.io/reference/markers/crd-validation), so I suspect it does nothing.